content/applications/productivity/spreadsheet/filters.rst

```diff
@@ -0,0 +1,205 @@
+- :guilabel:`Default value`: this selection appears **only** after a selection is made for the
+  :guilabel:`Related model` field. Using the drop-down menu, select the :guilabel:`Default value`.
+  The presented options are directly related to the selected :guilabel:`Related model`; changing the
+  :guilabel:`Related model` changes the options in the drop-down menu.
+
+.. image:: filters/relation.png
+   :align: center
+   :alt: The fields filled in for a relation filter to show only employees who left the company
+         because they retired.
+
+.. _filters/text:
+
+Text
+----
+
+This filter selection organizes and presents data in the spreadsheet that are based on specific
+words.
+
+First, the :guilabel:`Filter properties` must be configured. These properties determine what
+specific data is presented.
+
+Select this type of filter if the data should be filtered by specific words.
+
+Populate the following fields presented in the text filter:
+
+- :guilabel:`Label`: enter a name for the specific filter.
+- :guilabel:`Restrict values to a range`: tick the checkbox to limit the results to a specific
+  range of values. When enabled, a blank text box appears. Enter the range in the field. If the
+  entered range is not applicable, the box turns red and a red :icon:`fa-exclamation-circle`
+  :guilabel:`(Exclamation)` icon appears.
+- :guilabel:`Default value`: enter a default value for the search criteria in this field.
+- :guilabel:`Field matching`: the spreadsheet can be further filtered down, if desired. It is
+  possible to specify which fields on the spreadsheet to apply the filter on. Using the drop-down
+  menu, select the field that the filter applies to.
+
+  .. note::
+     The information presented in the spreadsheet is indicated in the :guilabel:`Field Matching`
+     section. For example, if a spreadsheet is presenting employee contracts organized by
+     department, the :guilabel:`Field Matching` section displays :guilabel:`Employees Analysis by
+     Department (Pivot #1)` in the field, above the blank lie where information is populated.
+
+Manage filters
+==============
+
+Once a filter is configured, it appears on the right-side of the spreadsheet.
+
+To change the order in which the filters appear, hover over a filter to reveal a
+:icon:`fa-ellipsis-v` :guilabel:`(vertical ellipses)` icon. Click on the icon and drag the filter to
+the desired location.
+
+To remove a configured filter, click the :icon:`fa-times` :guilabel:`(Clear)` icon to the right of
+the filter.
+
+To make any edits to the filter configuration, click the :icon:`fa-cog` :guilabel:`(Edit)` icon to
+the right of the :icon:`fa-times` :guilabel:`(Clear)` icon. Make any edits to the filter, then
+click :guilabel:`Save`.
+
+.. image:: filters/manage.png
+   :align: center
+   :alt: The icons displayed after filters are created and saved.
+
+All filters created for the spreadsheet appear in this list, until they are removes with the
```

Comment:
I think this last section should go above the image. I am not sure if it needs to be in a call out block, but when it is below the screenshot, it is easy to be lost at the end of the page.
```suggestion
All filters created for the spreadsheet appear in this list, until they are removed with the
```